scripts/ci/pylint_todo.txt

```diff
@@ -22,7 +22,7 @@
 ./airflow/contrib/hooks/sagemaker_hook.py
 ./airflow/contrib/hooks/segment_hook.py
 ./airflow/contrib/hooks/slack_webhook_hook.py
-./airflow/contrib/hooks/snowflake_hook.py
+./airflow/providers/snowflake/hooks/snowflake.py
```

Comment:
Please don't add the file here, better would be to fix the pylint issues instead. 👍

Reply:
removed

Reply:
Only snowflake hook is present in pylint_todo.txt. even though if we have a snowflake connector package. it is throwing `No name 'connector' in module 'snowflake'` error. 

<img width="1088" alt="Screenshot 2020-01-13 at 4 10 15 PM" src="https://user-images.githubusercontent.com/29749983/72249374-46ec5180-361f-11ea-87de-4de28708d399.png">

Reply:
If it is one specific error you cannot fix just disable it `pylint: disable=no-name-in-module` or `pylint: disable=no-member`

Reply:
At the end of the line